Patch written by Kurt Lust, UAntwerp.
Removes GNU-specific compiler options that are added automatically to those passed
via the regular CMake compiler flag options.
--- bowtie2-2.3.5.1/CMakeLists.txt.orig	2019-07-02 13:11:18.223165000 +0200
+++ bowtie2-2.3.5.1/CMakeLists.txt	2019-07-02 13:12:44.771437431 +0200
@@ -23,9 +23,9 @@
 set(USE_SRA 0)
 set(WITH_THREAD_PROFILING ${WITH_THREAD_PROFILING})
 set(POPCNT_CAPABILITY ${POPCNT_CAPABILITY})
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m64 -g3 -Wall -msse2")
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -funroll-loops")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE}")
 set(INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/bin" CACHE PATH "Installation directory for executables")
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS on)
